src/docs/product/integrations/gitlab/index.mdx

```diff
@@ -123,6 +123,31 @@ A `<SHORT-ID>` may look something like 'BACKEND-C' in the image below.
 
 ![Issue detail highlighting suspect commits](short-id.png)
 
+### Stack Trace Linking
```

Comment:
Please apply edits in previous file (as this content looks like it's duplicated here).